.github/workflows/ci.yml

```diff
@@ -111,6 +111,8 @@ jobs:
         uses: ScribeMD/docker-cache@0.5.0
         with:
           key: ddev-images-${{ runner.os }}-${{ steps.ddev_version.outputs.ddev_version }}-${{ hashFiles('.ddev/config.yaml', '.ddev/docker-compose.cypress.yaml', '.ddev/docker-compose.redis.yaml') }}
+          read-only: ${{ github.event_name == 'pull_request' }}
```

Comment:
What do you think of adding this for composer and npm cache as well? Is it even an option for the default cache action? I ask because I see similar duplicated caches for composer and npm for other branches as well. Since the hash is there, the cache should be fine across branches.

Reply:
Yes, it makes sense to ignore other caches as well, but no such option is available for the cache action. I see that there is an option `lookup-only`, which only checks if the cache exists and ignore restoring cache. however  it does not affect the behavior of cache saving. One approach could be to use this flag to check if the cache exists and then restore and save the cache using the restore and save actions. However, I'm still unsure if setting the `lookup-only` flag will disable cache saving, as the documentation does not indicate this.

> lookup-only - If true, only checks if cache entry exists and skips download. Does not change save cache behavior. Default: false

> Do you think I should create an issue on the cache action and see if I can raise a PR for this?

Reply:
> Do you think I should create an issue on the cache action and see if I can raise a PR for this?

Let's pass. Those cache sizes are typically not a problem. It would have been nice-to-have but we shouldn't get distracted.